code/game/objects/items/devices/radio/radio.dm

```diff
@@ -214,6 +214,7 @@ var/global/list/default_medbay_channels = list(
 		else
 			if (channels[chan_name] & FREQ_LISTENING)
 				channels[chan_name] &= ~FREQ_LISTENING
+				playsound(src.loc, null)

```

Comment:
I'm curious. `loc` over `src`. Wouldn't it be better to use `src` as the origin, so that you wouldn't cancel any other sounds that are played on the same time? And in any case, the src is the origin, so the src should be used as the origin.